.github/workflows/version.yml

```diff
@@ -11,6 +11,14 @@ jobs:
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
+       - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
```

Comment:
Do we have any significance of this id?

Reply:
Its the installation id of the `opensearch-trigger-bot` Github App.

Reply:
Is it fine to expose it or should we add it in secrets?

Reply:
Its fine. Its the same across all repos, we have been using it in [backport](https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/backport.yml#L22) and [dependabot](https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/dependabot_pr.yml#L18) workflows.

Reply:
Sounds good. Thanks for answering the questions.